doc/tutorial/alphabet/index.md

```diff
@@ -0,0 +1,334 @@
+- retrieve the alphabet size (seqan3::Alphabet::alphabet_size_v) and
+- compare their values (operators `==`, `>`, `>=`, ...).
+
+\note Of course, the seqan3::NucleotideAlphabet satisfies seqan3::Alphabet. <br/>
+      It actually extends the alphabet concept with the [complement()](\ref seqan3::NucleotideAlphabet::complement)
+      function.
+
+<br/>
+
+---
+
+# Other alphabets
+
+Until now, we have focused on alphabets for nucleotides to introduce the properties of SeqAn's alphabet
+on a specific example. SeqAn implements, however, many more alphabets. In this section, we want to give you
+an overview to the existing alphabets and in the end you can implement an additional one yourself.
+
+<br/>
+
+## The amino acid alphabet
+
+Proteins consist of one or more long chains of amino acids. The so-called primary structure of a protein is
+expressed as sequences over an amino acid alphabet. The seqan3::aa27 alphabet contains the standard one-letter code
+of the 20 canonical amino acids, as well as the two proteinogenic amino acids, a termination symbol and
+some wildcard characters. For details read the seqan3::Aminoacid page.
+
+<br/>
+
+## Structure and quality alphabets
+
+The alphabets for structure and quality are sequence *annotations*, as they describe additional
+properties of the respective sequence. 
+We distinguish three types:
+1. **Quality alphabet for nucleotides**. The values are produced by sequencing machines and represent the probability
+   that a nucleobase was recorded incorrectly. The characters are most commonly found in FASTQ files.
+   See seqan3::Quality for details.
+2. **RNA structure alphabets**. They describe RNA nucleobases as unpaired or up-/downstream paired and can be found
+   in annotated RNA sequence and alignment files (e.g. Stockholm format). Currently we provide the
+   [Dot Bracket](\ref seqan3::dot_bracket3) and [WUSS](\ref seqan3::wuss) formats.
+3. **Protein structure alphabet**. The [DSSP](\ref seqan3::dssp9) format represents secondary structure elements like
+   alpha helices and turns.
+
+You can build a [Cartesian Compositions](\ref seqan3::cartesian_composition) with a nucleotide and quality
+alphabet, or nucleotide / amino acid and structure alphabet that stores both information together.
+For the use cases just described we offer pre-defined composites (seqan3::qualified, seqan3::structured_rna,
+seqan3::structured_aa). See our API documentation for a detailed description of each.
+
+<br/>
+
+## Gap alphabet
+
+The seqan3::gap alphabet is the smallest alphabet in SeqAn, consisting of the gap character only.
+It is most often used in a [Union Composition](\ref seqan3::union_composition) with a nucleotide or amino acid alphabet
+to represent gapped sequences, e.g. in alignments. To create a gapped alphabet simply use seqan3::gapped<> with
+the alphabet type you want to extend.
+\snippet alphabet_main.cpp gapped
+
+<br/>
+
+## Create your own alphabet
```

Comment:
I think we should drop this section from the tutorial altogether as it really is not very easy and the alphabet tutorial should be the easiest possible. Let's focus only on using alphabets here.

Please make a backup of the text somewhere so we can integrate it into the concepts tutorial later on once we have simplified the alphabet concept.

Reply:
Really?? You asked me to spend several hours on re-doing the alphabet tutorial for discussing the concepts in more detail and now I should just drop it?!
Let's hope that the text is still useful later. I already have another branch with the enum interface, so I'll archive them together.